README.md

```diff
@@ -85,11 +85,11 @@ like this `'0 22 * * 1-5 America/Chicago'`.
 
 #### Natural-language formats
 
-Since sidekiq-cron `v1.7.0`, you can use the natural-language formats supported by Fugit, such as:
+Since Sidekiq-Cron `v1.7.0`, you can use the natural-language formats supported by Fugit, such as:
```

Comment:
the gem is called sidekiq-cron not Sidekiq-Cron, I would leave this as is

Reply:
ok! we have now a mix between Sidekiq-Cron (seems like the project name) and sidekiq-cron (the gem name) in the docs. I'm gonna revert this one, but in a near future will be nice to unify all references.